### PR TITLE
[BTRX-452][risk=no] Status refactoring

### PR DESCRIPTION
**Pull request by rushtong**
Small refactoring PR in my effort to make some progress on https://broadinstitute.atlassian.net/browse/BTRX-452

blocked on https://github.com/DataBiosphere/consent-ontology/pull/126

The goal with this set of stacked PRs is to clean up how we're using guice now so I can have a cleaner model to follow when updating consent. Consent will have some of the same problems we had here, but worse.
See also:
* https://github.com/DataBiosphere/consent-ontology/pull/124
* https://github.com/DataBiosphere/consent-ontology/pull/125
* https://github.com/DataBiosphere/consent-ontology/pull/126

### CODE REVIEW

File: src/main/java/org/broadinstitute/dsde/consent/ontology/resources/StatusResource.java [hunk 1]
```diff
@@ -2,6 +2,7 @@
 
 import com.codahale.metrics.health.HealthCheck;
 import com.codahale.metrics.health.HealthCheckRegistry;
+import com.google.inject.Inject;
```
Comment (tadeork) [inline src/main/java/org/broadinstitute/dsde/consent/ontology/resources/StatusResource.java:5]: Just a question to understand. Are we using dropwizard-guicer or google-guice? Or is just that dropwizard-guicer implements this last one?

**Comment by tadeork** [comment]
Looks good.

**Review by tadeork — APPROVED** [review]
Looks good

File: src/main/java/org/broadinstitute/dsde/consent/ontology/resources/StatusResource.java (hunk 1 above)
Comment (rushtong) [reply src/main/java/org/broadinstitute/dsde/consent/ontology/resources/StatusResource.java:5]: We are just using google-guice. At one point, we were using the dropwizard plugin, but somehow we never really implemented it. I'm fuzzy on the history here. The plugin does provide some more shortcuts, but I don't really think they are worth it and add a layer of abstraction over what Google does, which is nice in and of itself. 

My goal with this story (and set of PRs) is to clean up how we're using guice now so I can have a cleaner model to follow when updating consent. Consent will have some of the same problems we had here, but worse :-(

Comment (rushtong) [reply src/main/java/org/broadinstitute/dsde/consent/ontology/resources/StatusResource.java:5]: I'll add this summary to the PRs for better understanding.

Comment (rushtong) [reply src/main/java/org/broadinstitute/dsde/consent/ontology/resources/StatusResource.java:5]: Done, thanks.

**Pull request merged**